builders/deploy.yaml

```diff
@@ -18,22 +18,90 @@
 
 substitutions:
   _REGION:
-  _SERVICE:
+  _REPO: 'github.com/google/exposure-notifications-verification-server'
   _TAG:
 
 steps:
-- id: 'deploy'
-  name: 'gcr.io/google.com/cloudsdktool/cloud-sdk:303.0.0-alpine'
+#
+# adminapi
+#
+- id: 'deploy-adminapi'
+  name: 'gcr.io/google.com/cloudsdktool/cloud-sdk:307.0.0-alpine'
```

Comment:
Remove the version?

Reply:
Eh, two things:

1. I've been bitten by subtle changes in gcloud and would prefer to make deploys as reproducible as possible by reducing variables.

2. Cloud Build is going to cache the container image. So "latest" isn't actually latest anyway.

Reply:
Fair enough. I always prefer latest if possible because folks always forget to update, and the larger the diff on upgrade, the higher the chance of bug.